diff --git a/web/modules/contrib/social_media_links/src/Plugin/SocialMediaLinks/Platform/Linkedin.php b/web/modules/contrib/social_media_links/src/Plugin/SocialMediaLinks/Platform/Linkedin.php
index 660b6e7..e717d3e 100644
--- a/web/modules/contrib/social_media_links/src/Plugin/SocialMediaLinks/Platform/Linkedin.php
+++ b/web/modules/contrib/social_media_links/src/Plugin/SocialMediaLinks/Platform/Linkedin.php
@@ -10,7 +10,7 @@
  * @Platform(
  *   id = "linkedin",
  *   name = @Translation("LinkedIn"),
- *   urlPrefix = "https://www.linkedin.com/",
+ *   urlPrefix = "https://www.linkedin.com/in/",
  * )
  */
 class Linkedin extends PlatformBase {}
